### ISSUE TITLE
Unable to access new map's settings

### ISSUE THREAD

**Issue by WildStrawberryEVE** [exodus4d/pathfinder#562]
currently using 1.3.1.

Steps to reproduce:

1. Create new map with "+add" button. (does not matter - private or corporation)
2. Switch to new map
3. Try to access "settings" button from the right side Map menu

button does nothing - window with map settings does not appear.
No error in any log.

Button "settings" works well for older maps (created months ago).

**Comment by jajmo** [comment]
I can't reproduce - it works as expected for me.

Check your console output in developer tools when you click the button. Are there any errors?

**Comment by WildStrawberryEVE** [comment]
[![Screenshot from Gyazo](https://gyazo.com/c63ad6a5a71ae0e7515fdc7949235a4b/raw)](https://gyazo.com/c63ad6a5a71ae0e7515fdc7949235a4b)

`PATHFINDER v1.3.1
map_settings.js:163 Uncaught TypeError: Cannot read property 'length' of null
    at map_settings.js:163
    at Object.execCb (require.js:1)
    at q.check (require.js:1)
    at q.enable (require.js:1)
    at q.init (require.js:1)
    at require.js:1
(anonymous) @ map_settings.js:163
execCb @ require.js:1
check @ require.js:1
enable @ require.js:1
init @ require.js:1
(anonymous) @ require.js:1
setTimeout (async)
g.nextTick @ require.js:1
u @ require.js:1
requirejs @ require.js:1
t.fn.showMapSettingsDialog @ map_settings.js:77
(anonymous) @ page.js:660
dispatch @ jquery-3.1.1.min.js:3
g.handle @ jquery-3.1.1.min.js:3
trigger @ jquery-3.1.1.min.js:4
(anonymous) @ jquery-3.1.1.min.js:4
each @ jquery-3.1.1.min.js:2
each @ jquery-3.1.1.min.js:2
trigger @ jquery-3.1.1.min.js:4
t.fn.triggerMenuEvent @ page.js:432
(anonymous) @ page.js:330
dispatch @ jquery-3.1.1.min.js:3
g.handle @ jquery-3.1.1.min.js:3`

**Comment by WildStrawberryEVE** [comment]
OK, i think i found the problem - still not sure why did it happen.

First I manually expanded map table with:
ALTER TABLE map MODIFY COLUMN slackWebHookURL varchar(255);
Then i modified mapmodel.php so it would not try to revert it to 128 chars. I clicked "set up tables" but somehow column stayed "nullable".

So i fixed it with:
ALTER TABLE map MODIFY COLUMN slackWebHookURL varchar(255) NOT NULL;

Deleted problematic maps and now newly created maps don't have that problem.

I think we can close this one - Exodus is working on discord support out of box for 1.3.2 anyway :)
